df1-html/lib/Df1/Html/Render.hs

```diff
@@ -0,0 +1,137 @@
+{-# LANGUAGE OverloadedLists #-}
+{-# LANGUAGE OverloadedStrings #-}
+
+module Df1.Html.Render
```

Comment:
Excellent work with the documentation on this module!!!

Reply:
Thank you!! I now corrected the parsers that succeded with empty text.